app/views/shared/_header.html.erb

```diff
@@ -11,7 +11,7 @@
           <a href="#">Contact us</a>
         </li>
         <li class="header-links__link">
-          <a href="#">Get involved</a>
+          <a href="/en/get-involved">Get involved</a>
```

Comment:
A question rather than a suggestion. 
Just want to double check that we should be hard coding urls? Maybe @margOnline might be best to pass judgment on this? I know we are yet to hook up all the links to the pages, but just in case, we probably don't want this getting forgotten about?

Reply:
The best way of add links to the app is using the rails routes.
If you run `rails routes` you will see the name of the prefix of the urls plus with "_path" as suffix.
On the case above the link name is: `get_involved_path`.
So the code above will be: 

```
<%= link_to 'Get involved', get_involved_path %>  
```

Reply:
Since this is static data and not getting the data from a database or an api,  the only way to programmatically determine the url is to use translation files. I agree with @tomas-stefano's suggestion